lib/routes/user/marketplace/vendor_webview.dart

```diff
@@ -64,10 +64,80 @@ class VendorWebViewPageState extends State<VendorWebViewPage> {
     });
   }
 
+  _injectWebLNLibrary() {
+    String includeWebLN = "var webLNLib = document.createElement('script');\n" +
+        "webLNLib.src = 'https://unpkg.com/webln@0.2.0/dist/webln.min.js';\n" +
+        "webLNLib.integrity = 'sha384-mTReBqbhPO7ljQeIoFaD1NYS2KiYMwFJhUNpdwLj+VIuhhjvHQlZ1XpwzAvd93nQ';\n" +
+        "webLNLib.crossOrigin = 'anonymous';\n" +
+        "document.body.appendChild(webLNLib);";
+    _widgetWebview.evalJavascript(includeWebLN).whenComplete(() => _initializeWebLN());
+  }
+
+  _initializeWebLN() {
+    String initWebLN = "   var processedInvoices = [];\n" +
```

Comment:
I think it is good idea to place the script in its own resource file. Makes it better to maintain and read.